shared/js/background/classes/site.es6.js

```diff
@@ -105,14 +105,22 @@ class Site {
         return this.isProtectionEnabled() && this.enabledFeatures.includes(featureName)
     }
 
+    /**
+     * @param {import("@duckduckgo/privacy-grade/src/classes/trackers.js").TrackerData} t
```

Comment:
If the timing of approvals in privacy-grade is an issue we can change this to `@param {*}` for now.

Reply:
Made this change and will change it back once we land those changes.